app/views/users/show.html.erb

```diff
@@ -12,5 +12,11 @@
 <% if current_user && current_user.id == @user.id %>
   <%= link_to 'Edit', edit_user_path(@user) %> |
   <%= link_to 'Destroy', 'users#destroy' %> |
+<% else %>
+  <% if current_user && @user.id.to_s == FollowerFollowed.where(follower_id: current_user.id).pluck(:followed_id)[0] %>
+    <%= link_to 'Unfollow', "/follower_followed/#{@user.id}/destroy" %>
+  <% else %>
+    <%= link_to 'Follow', "/follower_followed/#{@user.id}/create" %>
+  <% end %>
```

Comment:
Here it is!  The reason for the PR.  You can only follow if you're logged in, and you can only unfollow if you've followed them.